workitem/expression_compiler.go

```diff
@@ -206,6 +206,24 @@ var DefaultTableJoins = func() TableJoinMap {
 			"typegroup.": res["typegroup"],
 		},
 	}
+
+	// Filter by parent's ID or human-readable Number
+	res["parent_link"] = &TableJoin{
+		TableName:  "work_item_links",
+		TableAlias: "parent_link",
+		// importing the link package here to get the link type is currently not
+		// possible because of an import cycle
+		On: Column("parent_link", "link_type_id") + "= '25C326A7-6D03-4F5A-B23B-86A9EE4171E9' AND " + Column("parent_link", "target_id") + "=" + Column(WorkItemStorage{}.TableName(), "id"),
```

Comment:
There can only be one parent of a work item by definition. So to me it makes no sense to join all possible links and then filter out the one (if any) we're interested in. This can be done in the ON part. When `parent` is used as a table alias throughout the query I want it to only reference this one particular parent and not any work item that is connected to the work item at hand.

Reply:
Yeah, sure :+1: